distribution/lib/Standard/Database/0.0.0-dev/src/Data/DB_Table.enso

```diff
@@ -633,7 +634,8 @@ type DB_Table
     filter_by_expression : Text -> Problem_Behavior -> DB_Table ! No_Such_Column | Invalid_Value_Type | Expression_Error
     filter_by_expression self expression on_problems=Report_Warning =
         column = self.evaluate_expression expression on_problems
-        self.filter column Filter_Condition.Is_True
+        result = self.filter column Filter_Condition.Is_True
+        Warning.attach (Deprecated.Warning "Standard.Database.Data.DB_Table.DB_Table" "filter_by_expression" "Deprecated: use `filter` with an `Expression` instead.") result
```

Comment:
I'd suggest 
```suggestion
        result = self.filter column Filter_Condition.Is_True
        Warning.attach (Deprecated.Warning "Standard.Database.Data.DB_Table.DB_Table" "filter_by_expression" "Deprecated: use `filter` with an `expr` instead.") result
```
as that is what the user would have to type.

Reply:
but you would choose Expression from the dropdown so felt that was correct?

Reply:
Ah! Ok I did not consider that.

Well in such case ignore my suggestion, current state seems OK.